AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.h

```diff
@@ -32,6 +33,11 @@
  */
 @property(nonatomic, readonly) NSString *status;
 
+/**
+ * Token status.
+ */
+@property(nonatomic, readonly) NSString *expiresOn;
```

Comment:
It's best practice to use `copy` for NSString properties.

Please fix all NSString properties in this class.